src/components/PostCard/PostCard.test.tsx

```diff
@@ -3,17 +3,19 @@ import PostCard from "./PostCard";
 import { Post } from "../../posts/types";
 
 describe("Given the PostCard component", () => {
-  describe("When rendered", () => {
-    test("Then it should show a post with a title 'Las arañas de ocho patas'", () => {
-      const expectedTitle = /Las arañas de ocho patas/i;
-      const post: Post = {
-        author: "",
-        content: " ",
-        date: 1,
-        id: "",
-        title: "Las arañas de ocho patas",
-      };
+  const post: Post = {
+    id: "",
+    author: "Macario",
+    content: "The spider has eight legs because it is a spider.",
+    date: new Date(1714751590000),
+    title: "Las arañas de ocho patas",
+    imageUrl: "imagendearaña.jpg",
+    alternativeText: "una araña",
+  };
 
+  describe("When it receives post with a title: 'Las arañas de ocho patas'", () => {
```

Comment:
when it receives a post with the title...

Reply:
hecho